packages/block-editor/src/components/date-format-picker/stories/index.story.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Internal dependencies
+ */
+import DateFormatPicker from '../';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+export default {
+	title: 'BlockEditor/DateFormatPicker',
+	component: DateFormatPicker,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'The `DateFormatPicker` component enables users to configure their preferred *date format*. This determines how dates are displayed.',
+			},
+		},
+	},
+	argTypes: {
+		defaultFormat: {
+			control: 'text',
+			description:
+				'The date format that will be used if the user selects "Default".',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		format: {
+			control: { type: null },
+			description:
+				'The selected date format. If `null`, _Default_ is selected.',
+			table: {
+				type: { summary: 'string' },
```

Comment:
```suggestion
				type: { summary: 'string | null' },
```

I think the `null` type is also allowed.